SQLQuery_ZooManagement.sql

```diff
@@ -0,0 +1,164 @@
+CREATE Database ZooManagement;
+
+Use ZooManagement;
+
+CREATE TABLE Species
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Species VARCHAR(150),
+SpeciesNumber INT,
+AnimalsNumber INT,

```

Comment:
The AnimalsNumber should not be stored inside the table because it means that every time a new animal leaves or joins the zoo a change is required here to update this value. It can be computed based on the entries there should be stored separately in an Animals table (the name of the animal, its age, etc.).